lib/keystore/keyring.go

```diff
@@ -62,15 +62,18 @@ type Sr25519Keyring struct {
 	KeyGeorge  *sr25519.Keypair
 	KeyHeather *sr25519.Keypair
 	KeyIan     *sr25519.Keypair
+
+	Keys []*sr25519.Keypair
 }
 
 // NewSr25519Keyring returns an initialized sr25519 Keyring
 func NewSr25519Keyring() (*Sr25519Keyring, error) {
 	kr := new(Sr25519Keyring)
+	kr.Keys = []*sr25519.Keypair{}
 
 	v := reflect.ValueOf(kr).Elem()
 
-	for i := 0; i < v.NumField(); i++ {
+	for i := 0; i < v.NumField()-1; i++ {
```

Comment:
Why are we iterating one less?

Reply:
the last field is an array of all the keys instead of `Alice`, `Bob`, etc

Reply:
Wouldn't it be better to have a conditional skip?
```
if v.Field(i).Name == "Keys" {
    continue
}
```